test/functional/inspec_exec_test.rb

```diff
@@ -53,7 +53,7 @@
   end
 
   it 'can execute the profile with a target_id passthrough' do
-    outpath = Dir.tmpdir
+    Dir.tmpdir
```

Comment:
I still don't see this being used - we're not in a tmpdir block.  Could this be removed entirely?